packages/core/src/components/Combobox/Combobox.tsx

```diff
@@ -115,6 +117,14 @@ export interface ComboboxProps extends VibeComponentProps {
   searchInputAriaLabel?: string;
   debounceRate?: number;
   searchInputRef?: React.RefObject<HTMLInputElement>;
+  /**
+   * Render additional action within the right section of search component.
+   */
+  renderAction?: React.ReactElement<typeof IconButton | typeof MenuButton>;
+  /**
+   * If true, hides the additional action when the user types in the search input.
+   */
+  hideRenderActionOnInput?: boolean;
```

Comment:
btw, did they requested this behavior?
if no, we don't have to supply them with that, to not overload the api with stuff that they don't necessarily need right now

Reply:
Meirav do want me to add this prop